lib/CLIManager.js

```diff
@@ -369,25 +384,47 @@ class CLIManager {
 
   /**
    * Handles any result from command execution.
-   * @param err
-   * @param result
+   * @param {CommandResult} result
    * @param [done]
    */
-  processCommandResult(err, result, done) {
+  processCommandResult(result, done) {
```

Comment:
About that... I've made `CommandResult` contain error and data because it was more convenient and because `processCommandResult` and the way it was being called were already breaking good practices. 
I imagine that during the refactoring command handlers will be made to pass error to callback as first argument and data - as second. I imagine that if a command fails, it would pass only error to callback as node.js practices dictate. `CommandResult` will probably change.

Reply:
Fair enough.